govtool/frontend/src/components/molecules/GovernanceActionsDatesBox.tsx

```diff
@@ -4,11 +4,12 @@ import { Trans } from "react-i18next";
 
 import { Tooltip, Typography } from "@atoms";
 import { useScreenDimension, useTranslation } from "@hooks";
+import { formatDisplayDate } from "@/utils";
 
 type GovernanceActionsDatesBoxProps = {
   createdDate: string;
-  expiryDate: string;
-  expiryEpochNo: number;
+  expiryDate: string | undefined;
```

Comment:
```suggestion
  expiryDate?: string;
```

Reply:
Right, but that way the `expiryDate` prop would not be required. And I think it's better if it's required but accepts undefined just for the types to match

Reply:
Agree with that